.ci/lint-python.sh

```diff
@@ -1,41 +1,32 @@
 #!/bin/sh
 
-DIR_TO_CHECK=${1}
-
 echo "running flake8"
 flake8 \
+    --config=./python-package/setup.cfg \
     --ignore=E501,W503 \
     --exclude=./.nuget,./external_libs,./python-package/build,./python-package/compile \
```

Comment:
Doesn't setting the config mean we can remove these two lines?

Reply:
ah yes! Missed this one, thank you. I'll push a commit in a bit removing them.

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/5872/commits/03e624bea6c4bfba8b531b40d124ec0ffd56eb67, thanks for catching that!